webtau-config/src/main/java/com/twosigma/webtau/cfg/WebTauConfig.java

```diff
@@ -180,6 +182,19 @@ public int maxRedirects() {
         return maxRedirects.getAsInt();
     }
 
+    public String getUserAgent() {
+        if (userAgent.isDefault()) {
+            return userAgent.getAsString();
+        }
+
+        String defaultValue = userAgent.getDefaultValue().toString();
+        return userAgent.getAsString() + "/" + defaultValue;
```

Comment:
would it look better to have the webtau/version in (), e.g.   partner-api-tests (webtau/1.8).
vs partner-api-tests/webtau/1.8, or what if people have a slash in their agent config? api/v2/webtau/1.8

I might be overthinking this.

Reply:
I think you might be right, I prefer it in () than after a /.